uh_internal/apps/dailyduties/utils.py

```diff
@@ -67,7 +68,7 @@ def get_printer_requests(self):
 
         return printer_requests
 
-    def get_voicemail(self, token):
```

Comment:
I'm thinking we should switch the token field to taking in the service. Then in `get_voicemail()` below you can just call the service that is passed in. Then we can call setup in ajax.py and pass it over instead of having to call setup twice.